portfolio/src/main/webapp/index.html

```diff
@@ -43,6 +43,18 @@ <h2 id="modalHeader">Modal Header</h2>
 
   <!--The Map-->
   <div id="map"></div>
+
+  <!--The Map form-->
+  <form id="mapForm" method="POST" action="/new-map-marker">
+    <p style="font-size:14px; margin-bottom:2px">Add your college to the map by submitting the form!</p>
+    <input type="text" maxlength="30" name="internName" placeholder="Your Name">
+    <br>
+    <input type="text" maxlength="30" name="collegeName" placeholder="College Name">
+    <br>
+    <input type="text" name="collegeAddress" placeholder="College Address">
```

Comment:
Consider having different boxes for different address components. Not sure how smart the API is, it might be looking for specific types of info.

Reply:
Tested and it still works without the zip or the state given